src/erlfmt.erl

```diff
@@ -409,7 +411,9 @@ equivalent({Type, _, L1, L2, L3}, {Type, _, R1, R2, R3}) ->
     equivalent(L1, R1) andalso equivalent(L2, R2) andalso equivalent(L3, R3);
 equivalent({Type, _, L1, L2, L3, L4}, {Type, _, R1, R2, R3, R4}) ->
     equivalent(L1, R1) andalso
-        equivalent(L2, R2) andalso equivalent(L3, R3) andalso equivalent(L4, R4);
+        equivalent(L2, R2) andalso
+        equivalent(L3, R3) andalso
+        equivalent(L4, R4);
```

Comment:
Looking at this change specifically, I'm actually not 100% convinced it's better now compared to what it was. I think we should reconsider how we handle things like this.

Reply:
I had a similar initial reaction, but I have being growing more and more fond of this solution after looking more and more at it and especially when considering our current inconsistent alternative.

Also one way to avoid this is for the user to group parts of the expression in parens to keep it on one line

Reply:
@michalmuskala proposed trying to rotate the associativity, so that this would break as:
```
equivalent(L1, R1) andalso equivalent(L2, R2) andalso equivalent(L3, R3) andalso
         equivalent(L4, R4)
```
which we hope is what most people would expect

Reply:
And I think this is worth trying

Reply:
Don't forget to test it with stuff like…

```erlang
AVariable andalso AnotherVariable andalso AnotherVariableMore …
```

```erlang
a_function:with_a(<<"very">>, Long, list, "of", <<"attributes">>) andalso a:short(one) andalso another:very_very_long_one() andalso …
```

```erlang
the:ever(funny) andalso begin blocks:of(code), that:appear(between), your:boolean(expressions) end andalso are:terrible(things, to, debug)
```

…and of course… [else-less ifs](https://medium.com/erlang-battleground/if-i-fell-in-love-with-you-f35cf6b80d15)… 🤦‍♂️ 
```erlang
is_registered(User)
  andalso password_matches(User, Password)
  andalso (has_active_session(User) orelse is_admin(User))
  andalso
    begin
      register_login(User, dates:now()),
      [ open_admin_session(User) || is_admin(User) ]
    end
```

Reply:
cool thank you for the test cases :)